src/test/java/net/eventsexpress/app/pages/Footer.java

```diff
@@ -16,4 +16,7 @@ public void scrollToFooter() {
         Base.scrollToElementXpath(FOOTER_XPATH);
     }
 
+    public void click_privacy_link(){Base.findElementByXpath(PRIVACY_LINK_XPATH).click();}
```

Comment:
code formatting

Reply:
correctly formatted the code as requested.